app/views/shared/_micropost_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_for @micropost do |f| %>
+  <%= render "shared/error_messages", object: f.object %>
+  <div class="field">
+    <%= f.text_area :content, placeholder: t("compose_new_post") %>
+  </div>
+  <%= f.submit t("post"), class: "btn btn-primary" %>
+   <span class="picture">
+    <%= f.file_field :picture, accept: "image/jpeg,image/gif,image/png" %>
+  </span>
+<% end %>
+<script type="text/javascript">
+  $('#micropost_picture').bind('change', function() {
```

Comment:
giải thích hàm này cho anh

Reply:
thằng #micropost_picture  gọi đến thằng có html có id micropost_picture để khi upload ảnh nên mà ảnh to hơn 5mb gọi thằng funtion bên dưới hiện 1 bảng alert ạ

Reply:
cụ thể là khi cái html có id là micropost_picture thay đổi (lúc upload ảnh lên thì gọi đến thằng funtion bên dưới , thằng funtion bên duới check xem file có lớn hơn 5MB không , nếu có thì hiện 1 bảng alert ạ